sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -38,6 +38,8 @@ let client = AzureCommunicationChatClient(baseUrl: baseUrl, authPolicy: authPoli
 
 A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat. Users also get near real-time updates for when others are typing and when they have read the messages.
 
+Users also get near real-time updates for when others are typing and when they have read the messages. To learn more, read about chat concepts [here](https://docs.microsoft.com/azure/communication-services/concepts/chat/concepts).
```

Comment:
word it as: 
Using the APIs, users can also send typing indicators when typing a message and read reciepts for the messages they have read in a chat thread. To learn more, read about chat concepts here.